inode.go

```
@@ -303,7 +317,16 @@ func (inode *Inode) Dump() {
 func (inode *Inode) DumpFlags(includeFalses bool) {
 	defer func() {
 		if state := recover(); state != nil {
-			panic(state)
+			var err error
+
+			switch state := state.(type) {
+			case error:
+				err = state
+			default:
+				err = fmt.Errorf("%v", state)
+			}
+
+			log.Panic(err)
```

Remove this.

---

That is needed because this function does not return an error. This should panic and rely on the calls to handle it according to the original design

---

Isn't one of the goals of modifying the library to remove the panic/recover pattern as much as possible?? If so, DumpFlags should also return an error, and callers will have to handle the error (and possibly propagate it up the stack).

---

I was actually trying to hold off on doing that until some more integration with gVisor. But I could make another PR soon ditching the entire panic/recovery model.